src/main/php/xp/web/Servers.class.php

```diff
@@ -15,19 +15,13 @@ public function newInstance($address, $arguments= []) {
         return new Standalone($address, new AsyncServer());
       }
     };
-    self::$SEQUENTIAL= new class(1, 'SEQUENTIAL') extends Servers {
```

Comment:
This should be reflected in https://github.com/xp-forge/web?tab=readme-ov-file#server-models